bitnami/kubernetes-event-exporter/templates/vpa.yaml

```diff
@@ -0,0 +1,56 @@
+{{- /*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if and (.Capabilities.APIVersions.Has "autoscaling.k8s.io/v1/VerticalPodAutoscaler") .Values.autoscaling.vpa.enabled }}
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
```

Comment:
As in other components, we include  k8's labels `component` and `part-of` in our vpa.yaml (in this case only the component one would proceed):

```
app.kubernetes.io/component: kubernetes-event-exporter
```

Reply:
Added `part-of` label. I've used [`fluent-bit`](https://github.com/bitnami/charts/blob/main/bitnami/fluent-bit/templates/vpa.yaml#L12)  chart for the reference and it doesn't have it either.

Reply:
Thank you! My bad, I said it the other way around. Indeed, is `part-of`. Component would reference a subset of the application, like when we have an application with `migrator, exporter, what-ever-component...`. Thanks for noticing!